src/server/tran_server.hpp

```diff
@@ -233,8 +235,8 @@ class tran_server
     int get_boot_info_from_page_server ();
     int reset_main_connection ();
 
-    int parse_server_host (const std::string &host);
-    int parse_page_server_hosts_config (std::string &hosts);
+    int create_connection_with_server_host (const std::string &host);
+    int create_connections_with_server_hosts_config (std::string &hosts);
```

Comment:
In this function, the connection_handler is created. And "parsing" doesn't indicate it. 
> Or how about making more appropriate name?

Actually, I struggled to come up with a proper name for this. I changed it several times and even though that "Should I change the code flow if I can't come up with a proper name?", but I concluded it would take more than enough. Do you happen to have a good name for this?  I'd love it if you could suggest it.

Reply:
I thinks that create_connection_handler() can take over the role of parse_server_host(). And other jobs like parsing and using host info(_with_server_host) doesn't need to show in the function name. Other option from me is make_connection_handler().

Reply:
How about [497194a](https://github.com/CUBRID/cubrid/pull/4464/commits/497194a6d7b48ac13011be64559c653cab32c84e)? I agree that we can hide parsing, but we already have `create_connectino_hander` function. They can be overloaded, but I don't want to do that since they have different return type and usage.

Another one I considered is 
create_connection_handlers() -> setup_connection_handlers(). Give me your thought.

Reply:
@wrlawodms off-line discussion was done.